components/ordered-list/src/stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ListItem from '@govuk-react/list-item';
+
+import OrderedList from '.';
+
+storiesOf('OrderedList', module).add('OrderedList default', () => (
+  <OrderedList>
+    <ListItem>OrderedList example</ListItem>
```

Comment:
it might be an idea to add a second `ListItem` to these examples so that one gets to better see how lists are rendered, including spacing between elements